README.md

```diff
@@ -149,6 +149,71 @@ b1$parent$stop()
 `b1$parent` is a `Chromote` object (as opposed to `ChromoteSession`), which represents the browser as a whole. This is explained in [The Chromote object model](#the-chromote-object-model).
 
 
+### Attaching to existing browsers and tabs
```

Comment:
I suggest moving this section below the "Chrome on remote hosts" section, since it's an uncommon use case, and some of the same stuff is covered there. Also, this should definitely mention the security issues with running your browser with the debugging port open -- it allows anyone to connect to that port without auth.

Reply:
D'oh! I missed that section. I integrated what I wrote with what was there, and added a security warning.